sbag/generators/javascript/templates/app/src/app/modules/__entity_name__/__entity_name__.module.ts.jinja

```diff
@@ -4,10 +4,16 @@ import { {{entity.name}}ListComponent } from './{{entity_name}}-list/{{entity_na
 import { {{entity.name}}Component } from './{{entity_name}}/{{entity_name}}.component';
 import { {{entity.name}}FormComponent } from './{{entity_name}}-form/{{entity_name}}-form.component';
 import { {{entity.name}}RoutingModule } from './{{  entity.name | lower}}-routing.module';
+import { CommonModule } from '@angular/common';
+import { NgxPaginationModule } from 'ngx-pagination';
```

Comment:
Do we need this here as well, since it's added to main module?

Reply:
It hasn't worked without import in those modules, but it can be removed from app.module.ts.
I've corrected that!